static/js/tools/map/time_slider.tsx

```diff
@@ -102,10 +102,10 @@ export function TimeSlider(props: TimeSliderProps): JSX.Element {
       index += 1;
       if (index === props.dates.length) {
         index = 0;
-        if (firstUpdate.current) {
-          firstUpdate.current = false;
+        if (!firstUpdate.current) {
+          setPlay(false);
```

Comment:
curious why we don't `setPlay(false)` if firstUpdate is true? maybe add some comments?

Reply:
Done